LFS171x/docs/introduction-to-hyperledger.md

```diff
@@ -1 +1,386 @@
+
+* Model your business blockchain network
+* Generate REST APIs for interacting with your blockchain network
+* Generate a skeleton Angular application.
+
+Built in Javascript, Hyperledger Composer provides an easy-to-use set of components that developers can quickly learn and implement. The project was contributed by Oxchains and IBM.
+
+## Hyperledger Composer (Simon Stone & Kathryn Harrison)
+[![Hyperledger Composer (Simon Stone & Kathryn Harrison)](../images/video-image.png)](https://youtu.be/91qtaojB9oI)
+[Some rights reserved](https://creativecommons.org/licenses/by-nc-nd/4.0/)
+
+## Hyperledger Composer (Continued)
+Hyperledger Composer has created a modelling language that allows you to define the assets, participants, and transactions that make up your business network using business vocabulary. In addition, the transaction logic is then written by developers using Javascript. This simple interface allows business people and technologists to work together on defining their business network.
+
+The [benefits](https://www.hyperledger.org/wp-content/uploads/2017/05/Hyperledger-Composer-Overview.pdf) of Hyperledger Composer are:
+
+* **Faster creation of blockchain applications**, eliminating the massive effort required to build blockchain applications from scratch
+* **Reduced risk with well-tested**, efficient design that aligns understanding across business and technical analysts
+* **Greater flexibility** as the higher-level abstractions make it far simpler to iterate.
+
+You can watch an introduction to Hyperledger Composer [here](https://www.youtube.com/watch?v=cNvOQp8r0xo).
+
+# Q/A with Brian Behlendorf, Executive Director of Hyperledger
+
+## Q/A with Brian Behlendorf, Executive Director of Hyperledger
+Before diving into Hyperledger Iroha, Hyperledger Sawtooth, and Hyperledger Fabric, we asked Brian Behlendorf, Executive Director of Hyperledger at The Linux Foundation, some questions. We hope these questions will give you a better perspective of the Hyperledger technologies for the upcoming technical chapters.
+
+## Reasons Why Developers Would Become Interested in Open Source Software
+[![Reasons Why Developers Would Become Interested in Open Source Software](../images/video-image.png)](https://youtu.be/oDsXhqqsYZM)
+[Some rights reserved](https://creativecommons.org/licenses/by-nc-nd/4.0/)
+
+## Hyperledger vs. Apache
+[![Hyperledger vs. Apache](../images/video-image.png)](https://youtu.be/LNfHx3DP1eE)
+[Some rights reserved](https://creativecommons.org/licenses/by-nc-nd/4.0/)
+
+## A Key Feature of Hyperledger Fabric, Hyperledger Sawtooth, and Hyperledger Iroha
+[![A Key Feature of Hyperledger Fabric, Hyperledger Sawtooth, and Hyperledger Iroha](../images/video-image.png)](https://youtu.be/hbIl1UQNws8)
+[Some rights reserved](https://creativecommons.org/licenses/by-nc-nd/4.0/)
```

Comment:
Remove - the course is actually licensed under https://creativecommons.org/licenses/by/4.0/. Unfortunately, EdX doesn't let you specify which Creative Common license for videos. Therefore, we will let the overall license cover these.